docs/sections/user_guide/yaml/components/orog.rst

```diff
@@ -0,0 +1,57 @@
+.. _orog_yaml:
+
+orog
+====
+
+Structured YAML to run the component ``orog`` is validated by JSON Schema and requires the ``orog:`` block, described below. If ``orog`` is to be run via a batch system, the ``platform:`` block, described :ref:`here <platform_yaml>`, is also required.
+
+Documentation for the UFS Utils ``orog`` program is :ufs-utils:`here <orog-gsl>`.
+
+Here is a prototype UW YAML ``orog:`` block, explained in detail below:
+
+.. highlight:: yaml
+.. literalinclude:: /shared/orog.yaml
+
+UW YAML for the ``orog:`` Block
+-------------------------------
+
+old_line1_items
+^^^^^^^^^^^^^^^
+
+Configuration parameters for the ``orog`` component corresponding to the first line entries prior to hash 57bd832 from (July 9, 2024). If using this section, a value for ``orog_file`` should also be provided.
+
+
+execution:
+^^^^^^^^^^
+
+See :ref:`here <execution_yaml>` for details.
+
+files_to_copy:
+^^^^^^^^^^^^^^
+
+See :ref:`this page <files_yaml>` for details.
+
+files_to_link:
+^^^^^^^^^^^^^^
+
+Identical to ``files_to_copy:`` except that symbolic links will be created in the run directory instead of copies.
+
+mask:
+^^^^^
+
+Boolean indicating whether only the land mask will be generated. Defaults to ".false."
```

Comment:
Should we highlight an instance of `mask: false` in the example `orog` YAML block? Is it required? If it defaults to `false` does it need to be included in the YAML or will it execute as `false` without an entry altogether?